message_sender/signals.py

```diff
@@ -16,3 +18,11 @@ def psh_fire_metrics_if_new(sender, instance, created, **kwargs):
             "metric_name": 'inbounds.created.sum',
             "metric_value": 1.0
         })
+
+
+def update_default_channels(sender, instance, created, **kwargs):
+    """ Post save hook to ensure that there is only one default
+    """
+    if instance.default:
+        Channel.objects.filter(default=True).exclude(
+            channel_id=instance.channel_id).update(default=False)
```

Comment:
As Rudi said "We probably also want to raise an error when setting a default=False on a default model, so that we can't end up in the state where we have no default models."
This might be a good place for it, or perhaps the admin `clean()` method